live-build/misc/ansible-roles/appliance-build.dcenter/tasks/main.yml

```diff
@@ -87,4 +88,10 @@
     regexp: '^preserve_hostname: false'
     line: 'preserve_hostname: true'
 
+# Update number of NFS threads to 64
```

Comment:
I'd suggest either removing the comment, or adding some commentary about why 64 is better than the default value of 8.